### PR TITLE
package-configurations: Add various versions of async packages

### PR DESCRIPTION
**Pull request by ShrutiDadoo**
Signed-off-by: Shruti Dadoo <shruti.dadoo@here.com>

### CODE REVIEW

**Review by fviernau — DISMISSED** [review]

File: package-configurations/NPM/async/3.2.1/vcs.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- pattern: "docs/**"
+  reason: "DOCUMENTATION_OF"
+- pattern: "logo/**"
+  reason: "DOCUMENTATION_OF"
+- pattern: "perf/**"
+  reason: "TEST_OF"
+- pattern: "support/**"
+  reason: "BUILD_TOOL_OF"
+- pattern: "test/**"
+  reason: "TEST_OF"
```
Comment (MarcelBochtler) [inline package-configurations/NPM/async/3.2.1/vcs.yml:17]: Missing empty line at the end of the file.

File: package-configurations/NPM/async/3.2.2/vcs.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- pattern: "docs/**"
+  reason: "DOCUMENTATION_OF"
+- pattern: "logo/**"
+  reason: "DOCUMENTATION_OF"
+- pattern: "perf/**"
+  reason: "TEST_OF"
+- pattern: "support/**"
+  reason: "BUILD_TOOL_OF"
+- pattern: "test/**"
+  reason: "TEST_OF"
```
Comment (MarcelBochtler) [inline package-configurations/NPM/async/3.2.2/vcs.yml:17]: Missing empty line at the end of the file.

File: package-configurations/NPM/async/3.2.3/vcs.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+- pattern: "docs/**"
+  reason: "DOCUMENTATION_OF"
+- pattern: "logo/**"
+  reason: "DOCUMENTATION_OF"
+- pattern: "perf/**"
+  reason: "TEST_OF"
+- pattern: "support/**"
+  reason: "BUILD_TOOL_OF"
+- pattern: "test/**"
+  reason: "TEST_OF"
```
Comment (MarcelBochtler) [inline package-configurations/NPM/async/3.2.3/vcs.yml:17]: Missing empty line at the end of the file.

File: package-configurations/NPM/async/3.2.4/vcs.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+- pattern: "docs/**"
+  reason: "DOCUMENTATION_OF"
+- pattern: "logo/**"
+  reason: "DOCUMENTATION_OF"
+- pattern: "perf/**"
+  reason: "TEST_OF"
+- pattern: "support/**"
+  reason: "BUILD_TOOL_OF"
+- pattern: "test/**"
+  reason: "TEST_OF"
```
Comment (MarcelBochtler) [inline package-configurations/NPM/async/3.2.4/vcs.yml:17]: Missing empty line at the end of the file.

**Review by MarcelBochtler — CHANGES_REQUESTED** [review]

**Review by tsteenbe — DISMISSED** [review]

**Review by fviernau — APPROVED** [review]

**Review by fviernau — APPROVED** [review]

**Pull request merged**